pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -99,10 +99,12 @@
 
 , disabledTestPaths ? []
 
+# Allow passing in a custom stdenv to buildPython*
+, stdenv ? python.stdenv
```

Comment:
We need to test first whether this does not break cross.

Reply:
Could it break cross for anything other than `tensorflow`?

Reply:
How do we test this? 😆

Reply:
Well, I checked and sure enough, overriding `buildPythonPackage` wasn't necessary. Reverted. Just `icu`, and it still works